### DISCUSSION TITLE
Google pixel weather color not matching

### DISCUSSION THREAD

**Discussion by akrram09** [breezy-weather/breezy-weather#1119] [Ideas]
### Describe your idea

It's cool idea to add the pixel weather widget for breezy weather users but the widgets are not same material color 3 shade
* in dark mode they match but light mode not*
The one in the top in breezy weather
![Screenshot_2024-06-10-23-24-27-75_67d7f8591eff9881972c1169b8bca2d8](https://github.com/breezy-weather/breezy-weather/assets/140515513/98433167-2400-44b0-9563-bb546ac2743c)
![Screenshot_2024-06-10-23-24-39-26_67d7f8591eff9881972c1169b8bca2d8](https://github.com/breezy-weather/breezy-weather/assets/140515513/4d42ffd7-d8c5-40c0-bea6-a1741fb9e5f6)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Duplicate of #316
